src/app/business/project/[project_id]/page.tsx

```diff
@@ -160,6 +186,12 @@ export default function Dashboard() {
                           </div>
                         </div>
                       </CardContent>
+                      <Button
+                        className="w-full sm:w-[50%] md:w-[30%] lg:w-[15%]"
+                        onClick={handleCompleteProject}
+                      >
+                        Update Status
```

Comment:
If sole purpose of this is to set status as completed then update text 'Update Status' to reflect it is supposed to set project as completed, current text 'Update Status' misleads into thinking there are more than one status to set here